src/js/components/organisms-page/resellers/resellers.js

```diff
@@ -32,6 +43,41 @@ export default {
         },
         showAllAuthorizedResellers() {
             this.visibleAllAuthorizedResellers = true;
+        },
+        newRow(el) {
+            const rect = el.getBoundingClientRect(),
+                index = el.dataset.index - this.showMax;
+
+            cardsYPos.push(rect.top);
+
+            if (cardsYPos.length > 1) {
+                return cardsYPos[index - 1] !== cardsYPos[index];
+            }
+
+            return false;
+        },
+        setDelay(el) {
+            if (this.newRow(el)) {
+                rowIndex++;
+            }
+
+            return rowIndex * delay / 1000;
+        },
+        beforeEnter(el) {
+            TweenLite.set(el, {opacity: 0, y: resellerCardOffset});
+        },
+        enter(el, done) {
+            const transition = TweenLite.fromTo(el, resellerCardTransitionDuration, {
+                opacity: 0,
+                y: resellerCardOffset
+            }, {
+                opacity: 1,
+                y: 0,
+                ease: Power3.easeOut,
+                onComplete: done
+            });
+
+            transition.delay(this.setDelay(el));
```

Comment:
You can use the 5th parameter of the fromTo function to set the delay, so line 78 could be:
 }, this.setDelay(el));

See: https://greensock.com/docs/TweenLite/static.fromTo